apps/sms/lib/bridge/plugins/dispose/service.js

```diff
@@ -0,0 +1,40 @@
+!((define)=>{define((require,exports,module)=>{
+
+  /**
+   * Mini Logger
+   *
+   * @type {Function}
+   */
+
+  var debug = 1 ? function(arg1, ...args) {
+    var type = `[${self.constructor.name}][${location.pathname}]`;
+    console.log(`[DisposableService]${type} - "${arg1}"`, ...args);
+  } : () => {};
+
+  module.exports = function(service) {
+    debug('attaching plugin');
+
+    service.method('@@requestDisposal', (clientId) => {
+      debug('Client %s is requesting to dispose service', clientId);
+
+      let canBeDisposed = true;
+      service.eachClient((client) => {
+        if (client.id !== clientId) {
+          canBeDisposed = false;
+        }
```

Comment:
Mmm, nope, but if we decide to move this forward we'll propose better ```eachClient``` to Wilson :)